client/memory.js

```diff
@@ -79,12 +79,13 @@ export default class Memory extends React.Component {
       }
     }).then(function(res) {
       var memory = JSON.parse(res['_bodyInit']);
+      console.log(memory);
       var microsoftTags = [];
       var clarifaiTags = [];
-      if (memory.analyses[0] === null) {
+      if (memory.analyses[0].tags && memory.analyses.tags.length > 0) {
         microsoftTags = memory.analyses[0].tags;
```

Comment:
also, heads up, the AI API calls are not made synchronously to reduce the latency of the /api/memories/upload call. So there might be scenarios where the Msft API gets returned first and gets pushed into the analyses array first